sdk/iot/Azure.Iot.Hub.Service/src/IoTHubServiceClient.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Iot.Hub.Service
+{
+    /// <summary>
+    /// The IoT Hub Service Client
+    /// </summary>
+    public class IoTHubServiceClient
+    {
+        /// <summary>
+        /// place holder for Devices
+        /// </summary>
+        public DevicesClient Devices;
+        /// <summary>
```

Comment:
Are we not going to have query at this level?

Reply:
That's a good one to discuss. We decided CL grouping for twins under Devices and Modules as per onenote. Do we want to keep Query as a separate client instead?

Reply:
yes that is right - [groupings](https://microsoft.sharepoint.com/teams/IotHub22/_layouts/OneNote.aspx?id=%2Fteams%2FIotHub22%2FSiteAssets%2FIot%20Hub%20Notebook&wd=target%28Hub%20Track2.one%7CC5D8B275-CA89-4E84-A686-F013BD35B146%2FProposed%20groupings%20of%20client%7CB26F8212-BDF9-407A-BFCD-01BC6E5C71E1%2F%29
onenote:https://microsoft.sharepoint.com/teams/IotHub22/SiteAssets/Iot%20Hub%20Notebook/Hub%20Track2.one#Proposed%20groupings%20of%20client&section-id={C5D8B275-CA89-4E84-A686-F013BD35B146}&page-id={B26F8212-BDF9-407A-BFCD-01BC6E5C71E1}&end)

Reply:
Well query spans over several areas i.e not just twins but also jobs and also raw queries. What about those?

Reply:
Jobs queries are under jobs grouping.

Reply:
This can be changed later if this approach we decided does not work well. We thought Query for each individual sub client would be more discoverable.

Reply:
What is raw queries?